docs/user-guide/how_to_use_svc_plugin.md

```diff
@@ -0,0 +1,332 @@
+        WORKER_HOST=`cat /etc/volcano/worker.host | sed 's/$/&:2222/g' | sed 's/^/"/;s/$/"/' | tr "\n" ","`;
+        export TF_CONFIG={\"cluster\":{\"ps\":[${PS_HOST}],\"worker\":[${WORKER_HOST}]},\"task\":{\"type\":\"ps\",\"index\":${VK_TASK_INDEX}},\"environment\":\"cloud\"};
+        python /var/tf_dist_mnist/dist_mnist.py
+    env:
+    - name: VK_TASK_INDEX
+      value: "0"
+    - name: VC_TASK_INDEX
+      value: "0"
+    - name: VC_PS_HOSTS     ## Environment variable `VC_PS_HOSTS` contains the domains of all the `ps` hosts. 
+      valueFrom:
+        configMapKeyRef:
+          key: VC_PS_HOSTS
+          name: tensorflow-dist-mnist-svc
+    - name: VC_PS_NUM       ## Environment variable `VC_PS_NUM` contains the number of `ps` hosts.
+      valueFrom:
+        configMapKeyRef:
+          key: VC_PS_NUM
+          name: tensorflow-dist-mnist-svc
+    - name: VC_WORKER_HOSTS   ## Environment variable `VC_WORKER_HOSTS` contains the domains of all the `worker` hosts.
+      valueFrom:
+        configMapKeyRef:
+          key: VC_WORKER_HOSTS
+          name: tensorflow-dist-mnist-svc
+    - name: VC_WORKER_NUM     ## Environment variable `VC_WORKER_NUM` contains the number of `worker` hosts.
+      valueFrom:
+        configMapKeyRef:
+          key: VC_WORKER_NUM
+          name: tensorflow-dist-mnist-svc
+    image: volcanosh/dist-mnist-tf-example:0.0.1
+    name: tensorflow
+    ports:
+    - containerPort: 2222
+      name: tfjob-port
+      protocol: TCP
+    resources: {}
+    volumeMounts:   ## Mount the configmap generated for the job under `/etc/volcano`, which contains all host files. 
+    - mountPath: /etc/volcano
+      name: tensorflow-dist-mnist-svc
+    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      name: kube-api-access-wflz5
+      readOnly: true
+  dnsPolicy: ClusterFirst
+  enableServiceLinks: true
+  hostname: tensorflow-dist-mnist-ps-0    ## Add `hostname` filed
+  nodeName: volcano-control-plane
+  restartPolicy: Never
+  schedulerName: volcano
+  subdomain: tensorflow-dist-mnist        ## Add `subdomain` filed
+  tolerations:
+  - effect: NoExecute
+    key: node.kubernetes.io/not-ready
+    operator: Exists
+    tolerationSeconds: 300
+  - effect: NoExecute
+    key: node.kubernetes.io/unreachable
+    operator: Exists
+    tolerationSeconds: 300
+  volumes:
+  - configMap:    ## Configmap generated for the job
+      defaultMode: 420
+      name: tensorflow-dist-mnist-svc
+    name: tensorflow-dist-mnist-svc
+status:
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: "2022-04-13T02:08:17Z"
+    status: "True"
+    type: Initialized
+  - lastProbeTime: null
+    lastTransitionTime: "2022-04-13T02:08:18Z"
+    status: "True"
+    type: Ready
+  - lastProbeTime: null
+    lastTransitionTime: "2022-04-13T02:08:18Z"
+    status: "True"
+    type: ContainersReady
+  - lastProbeTime: null
+    lastTransitionTime: "2022-04-13T02:08:17Z"
+    status: "True"
+    type: PodScheduled
```

Comment:
A little bit distracting for me, I would suggest deleting all unrelated fields including but not limited to this block.

Reply:
it's better remove the unrelated section like status to make the example simple and clean